module/motion/KinematicsConfiguration/data/config/KinematicsConfiguration.yaml

```diff
@@ -27,7 +27,7 @@ head:
   # head movement limits
   limits: # pitch limit is a function of yaw ... need to calculate this properly
```

Comment:
If something needs to be changed here, it needs to be done before this PR is merged.